packages/formatting/src/utils/validation.ts

```
@@ -0,0 +1,153 @@
+  const result = PresetNameSchema.safeParse(name);
+  if (result.success) {
+    return result.data;
+  }
+  // Default to standard if invalid
+  return 'standard';
+}
+
+/**
+ * Validate formatter types
+ */
+export function validateFormatterTypes(types: string[]): SetupOptions['formatters'] {
```

ESLint

[error] Array type using 'string[]' is forbidden. Use 'Array<string>' instead.

(@typescript-eslint/array-type)